bitnami/dataplatform-bp1/README.md

```diff
@@ -0,0 +1,193 @@
+# Data Platform Blueprint 1 with Kafka-Spark-Solr
+
+Enterprise applications increasingly rely on large amounts of data, that needs be distributed, processed, and stored.
```

Comment:
Is this README following the same format as the rest? The TL;DR section, requirements and other sentences

Reply:
More or less yes.
The only section that differs is the description since it is indicating what the dataplatform is and the components it bundles because there is not an official website yet where that information is provided.